timescaledb/how-to-guides/compression/decompress-chunks.md

```diff
@@ -4,6 +4,10 @@ the chunk first. Inserting data into a compressed chunk is more computationally
 expensive than inserting data into an uncompressed chunk, so decompressing the
 chunk is also a good idea if you need to backfill large amounts of data.
 
+If you are using TimescaleDB 2.3 or later, decompression is automated when you
+ingest older data. For earlier versions of TimescaleDB, you need to perform
+decompression manually.
```

Comment:
I'm not 100% sure this is the best way to state this. @svenklemm or @erimatnor. Thoughts?

Reply:
There's a somewhat related issue in https://github.com/timescale/docs/issues/168 and I've pinged @gayyappan on Slack to ask for assistance. I think we might need to do some more thinking on this section.

Reply:
It was originally a comment from @jonatas but I might have misunderstood. Is there a better place to put this note? Or should the note read differently?